dali/pipeline/operator/op_schema.h

```diff
@@ -388,6 +411,75 @@ class DLL_PUBLIC OpSchema {
 
   DLL_PUBLIC string Dox() const;
 
+  DLL_PUBLIC bool CanUseDefaultInputDox() {
+    // TODO(klecki): add a flag to disallow this
+    if (MinNumInput() == 1 && MaxNumInput() == 1 && input_docs_.size() == 0) {
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * @brief Check if this operator has input documentation specified
+   *
+   * If this is true `GetCallSignatureInputs` and `InputDox` can be used to obtain it
+   *
+   * @return DLL_PUBLIC HasInputDox
+   */
+  DLL_PUBLIC bool HasInputDox() {
+    if (static_cast<int>(input_docs_.size()) == MaxNumInput()) {
+      return true;
```

Comment:
This looks like a silent error. Moreover, what about operators that can potentially take sh**load of inputs - i.e. something like tensor concatenation? How to document just some of the inputs? What it's going to look like for such variadic operators? I think it makes sense to produce something like
`__call__(Input0, Input1, ... , axis = 0)`

Reply:
This would be a case for custom docstring. I intend to add such option.